server.js

```diff
@@ -4222,6 +4223,43 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// ImageLayers.io integration.
+camp.route(/^\/imagelayers\/([^\/]+)\/([^\/]+)\/([^\/]*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];
+  var repo = match[2];
+  var tag = match[3]
+  var format = match[4];
+  if (user === '_') {
+    user = 'library';
+  }
+  var path = user + '/' + repo;
+  var badgeData = getBadgeData('imagelayers', data);
+  var options = {
+    method: 'POST',
+    json: {
+      "repos": [{"name": path, "tag": tag}]
+    },
+    uri: 'https://imagelayers.io:8888/registry/analyze'
+  };
+  request(options, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      size = filesize(buffer[0].repo.size, {round: 0, base: 10});
```

Comment:
I wasn't sure how to use metric to get a file size by looking at the function. But it looks like maybe I can just add a "B" to the line. I'll try this.